scripts/build.d/qt

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "${SYNCGIT}" ]; then
+  pkgname=qt
+  pkgmajorver=${MAJOR_VER:-6.4}
+  pkgsubver=${SUB_VER:-1}
+  pkgver=$pkgmajorver.$pkgsubver
+  pkgmd5=${MD5:-4694f47f13f460965d68c17d6e277730}
+  pkgfull=$pkgname-$pkgver
+  pkgfoler=qt-everywhere-src-$pkgver
+
+  pkgfn=$pkgfull.zip
+  pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.zip
+
+  download_md5 $pkgfn $pkgurl $pkgmd5
+
+  mkdir -p ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
+  pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src > /dev/null
+    unzip ${DEVENVDLROOT}/$pkgfn
+    mv $pkgfoler $pkgfull
+  popd
+else
+  pkgname=qt
+  pkgbranch=${VERSION:-6.4.1}
+  pkgfull=$pkgname-$pkgbranch
+  syncgit git://code.qt.io/qt qt5 ${pkgbranch} ${pkgfull}
```

Comment:
Is `qt5` expected?

Reply:
Yes, qt5 is expected!
[official doc](https://wiki.qt.io/Building_Qt_6_from_Git#Getting_the_source_code)

Reply:
Oh, it looks so much like a bug.  @j8xixo12 perhaps you can add the link in the code comment to make it obvious for maintainers.

Reply:
+ 1 for adding comments, and then we can land the code.

Reply:
@yungyuc @tai271828 I just added comments for qt repo name.